src/Converter/Extensions/InstagramConverter.php

```diff
@@ -0,0 +1,71 @@
+<?php
+
+namespace Predmond\HtmlToAmp\Converter\Extensions;
+
+use League\Event\EventInterface;
+use Predmond\HtmlToAmp\ElementInterface;
+use Predmond\HtmlToAmp\Converter\ConverterInterface;
+
+class InstagramConverter implements ConverterInterface
+{
+    public function hasInstagram(ElementInterface $element)
+    {
+        $classes = explode(' ', $element->getAttribute('class'));
+        return in_array('instagram-media', $classes);
```

Comment:
Can you also add a check for the attribute `data-instgrm-version` as well? Both checks are good, and the `data-instgrm-version` seems to be a staple. You don't have to check the value, but you could do something like this:

```php
array_key_exists('data-instgrm-version', $element->getAttributes());
```